### ISSUE TITLE
[Feature] Blood Transferable 

### ISSUE THREAD

**Issue by U-R-E** [TeamLapen/Vampirism#857]
I was wondering if a feature that allows hoppers to transfer blood from sieves, grinders, and blood containers would be a possibility to implement or even a "blood transfer pipe" maybe. 

Having bottles are nice for long trips or adventuring around the map, but having a way to automatically transfer the blood from at least container to container would be nice without have to click a bottle back and forth This would also help balance the game a slight bit more since regular food for hunters can be automated in a way that they just have to pickup the cooked steak and keep going whereas Vampires must spend time moving it all around manually.

**Comment by maxanier** [comment]
Have you tried some other mod that introduces pipes?
In theory any pipe that supports generic fluids should work with blood containers as well.
However, I have not tested this yet.

**Comment by U-R-E** [comment]
> Have you tried some other mod that introduces pipes?
> In theory any pipe that supports generic fluids should work with blood containers as well.
> However, I have not tested this yet.

Hey just wanted to update. Yes, I tried what you suggested and it worked perfectly. I used a mod called RefinedPipes since it was a super lightweight mod (173KB) incase anyone else is wondering. I was able to let the blood flow from a grinder/sieve machine into another container for the blood pedestal. It makes charging weapons/etc up easier and faster.
